smart_campus/app/templates/app/beacon_edit_page.html

```diff
@@ -0,0 +1,71 @@
+{% extends 'app/base.html' %}
+{% load static %}
+
+{% block stylesheet %}
+  {{ block.super }}
+  <link rel="stylesheet" type="text/css" href="{% static "app/css/edit.css" %}">
+{% endblock %}
+
+{% block content %}
+  <div class="ui text container">
+    <form class="ui big borderless form" action="" method="POST" id="beacon_form">
+      {% csrf_token %}
+      <div class="required five wide field">
+        <label>Beacon ID</label>
+        <input type="text" name="beacon_id" placeholder="801-xxxxx-xxxxx" value="{{ form_data.beacon_id }}" readonly="">
+      </div>
+      <div class="required five wide field">
+        <label>名稱</label>
+        <input type="text" name="name" value="{{ form_data.name }}">
+      </div>
+      <div class="field">
+        <label>描述</label>
+        <textarea  rows="2" form="beacon_form" type="text" name="description">{{ form_data.description }}</textarea>
```

Comment:
Remove additional spaces between `textarea` and `row`.

Reply:
Removed it.